content/200-concepts/100-components/02-prisma-client/035-select-fields.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: 'Select fields'
```

Comment:
I think this page should highlight a couple of more things:

- examples for `include`
- examples for nested combos of `select` and `include`